process.py

```diff
@@ -150,5 +156,146 @@ def __init__(self, **kwargs):
             self.log_output,
             ], **kwargs)
 
+    @traitlets.observe('process')
+    def _observe_process(self, change):
+        self.update()
+
     def update(self):
         self.progress_bar.update()
+
+
+class WorkChainSelector(ipw.HBox):
+
+    # The PK of a 'aiida.workflows:quantumespresso.pw.bands' WorkChainNode.
+    value = traitlets.Int(allow_none=True)
+
+    # When this trait is set to a positive value, the work chains are automatically
+    # refreshed every `auto_refresh_interval` seconds.
+    auto_refresh_interval = traitlets.Int(allow_none=True)  # seconds
```

Comment:
will it work if you set it to `None`?

Reply:
I don't think so, good catch!
```suggestion
    auto_refresh_interval = traitlets.Int()  # seconds
```